src/core/agent/services/identifierService.ts

```diff
@@ -328,18 +328,115 @@ class IdentifierService extends AgentService {
       (identifier: IdentifierResult) =>
         !storageIdentifiers.find((item) => identifier.prefix === item.id)
     );
-    if (unSyncedData.length) {
-      //sync the storage with the signify data
-      for (const identifier of unSyncedData) {
+    
+    const [unSyncedDataWithGroup, unSyncedDataWithoutGroup] = [
+      unSyncedData.filter((item: HabState) => item.group !== undefined),
+      unSyncedData.filter((item: HabState) => item.group === undefined),
+    ];
+
+    if (unSyncedDataWithoutGroup.length) {
+      for (const identifier of unSyncedDataWithoutGroup) {
+        if (identifier.name.startsWith("XX")) {
+          continue;
+        }
+
+        const op = await this.props.signifyClient.operations().get(`done.${identifier.prefix}`)
```

Comment:
OK, so for local group members this will be `done` but after my next PR for individual identifiers they will be `witness`. So we could have `witness` or `done`.

Can we check witness first, and then done if witness does not exist?